fast_api_als/database/db_helper.py

```diff
@@ -65,7 +71,7 @@ def insert_oem_lead(self, uuid: str, make: str, model: str, date: str, email: st
         }
 
         res = self.table.put_item(Item=item)
-
+        get_code(res, "put_item")
```

Comment:
which of these two is better qualified to be used as operation arg:
"put_item | query" or function's name?

Reply:
In my opinion, "put_item | query"(action done) is a better option. Function remains same for most part , irrespective of action done. If there are major differences in logging information, then we might consider function's name. But in this context, "put_item | query"(action done) should be enough

Reply:
how would you out figure which function's put_item got logged?

Reply:
Before, I misunderstood the question. Using the functions name would be better idea (if only one has to choosen from function name and action(put_item etc) ) to function's name. I have modified the code to include the function name along the action done.